ui/src/app/table/select-all/select-all.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, ElementRef, EventEmitter, Input, Output, ViewChild} from "@angular/core"
+import {SvgHelper, SvgIcon} from "../../core/SvgHelper"
+import {SelectAllEvent} from "../../models"
+
+export enum SelectAll {
+  SELECT_ALL,
+  SELECT_ALL_ON_PAGE
```

Comment:
Thinking that we should change the name of "SELECT_ALL_ON_PAGE" to "SELECT_PAGE"